src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -205,7 +211,7 @@ Record CondTraverseConsume(OpBase *opBase) {
 										  &op->edges);
 		}
 
-		_CondTraverse_SetEdge(op, op->r);
+		if(!_CondTraverse_SetEdge(op, op->r)) return NULL;
```

Comment:
How's it possible for _CondTraverse_SetEdge to fail at this point?

Reply:
You're right, I think this is unnecessary.